sourcefiles/fastmagic.py

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
If this is replacing the IPS file then the IPS file should be deleted.

Reply:
Will do.